pkg/adapter/containers.go

```diff
@@ -90,7 +90,7 @@ func (r *LocalRuntime) StopContainers(ctx context.Context, cli *cliconfig.StopVa
 	}
 
 	ctrs, err := shortcuts.GetContainersByContext(cli.All, cli.Latest, names, r.Runtime)
-	if err != nil {
+	if err != nil && !(cli.AllowMissing && errors.Cause(err) == define.ErrNoSuchCtr) {
```

Comment:
Can we log, instead of ignoring, errors here?

Reply:
Sure can do. Note that this behavior predates the PR.

Reply:
Now logged in Get{Containers,Pods}ByContext